### PR TITLE
Add 1.28 Tinkerbell E2E

### PR DESCRIPTION
**Pull request by tatlat**
*Issue #, if available:*

*Description of changes:*

*Testing (if applicable):*

*Documentation added/planned (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/e2e/tinkerbell_test.go [hunk 1]
```diff
@@ -412,6 +428,22 @@ func TestTinkerbellKubernetes127BottleRocketCuratedPackagesPrometheusSimpleFlow(
 	runCuratedPackagesPrometheusInstallTinkerbellSimpleFlow(test)
 }
 
+func TestTinkerbellKubernetes127UbuntuCuratedPackagesClusterAutoscalerSimpleFlow(t *testing.T) {
```
Comment (mitalipaygude) [inline test/e2e/tinkerbell_test.go:431]: we are adding this for 1.27?

**Review by cxbrowne1207 — APPROVED** [review]

**Review by mitalipaygude — APPROVED** [review]

File: test/e2e/tinkerbell_test.go (hunk 1 above)
Comment (tatlat) [reply test/e2e/tinkerbell_test.go:431]: yeah, jonathan suggested it

**Pull request merged**